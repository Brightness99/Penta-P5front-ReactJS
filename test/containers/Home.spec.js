import React from 'react';
import { mount } from 'enzyme';

import Home from 'containers/Home';

jest.mock('react-router-dom/NavLink', () =>
  (({ to, children }) => (<a href={to}>{children}</a>))
);

describe('Home', () => {
  const wrapper = mount(<Home />);

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
