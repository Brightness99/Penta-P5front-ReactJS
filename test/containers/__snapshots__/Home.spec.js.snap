// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render properly 1`] = `"<div><a href=\\"/configuracao-catalogo\\">Home Page</a></div>"`;
